docs/modules/ROOT/pages/governance.adoc

```diff
@@ -119,7 +119,7 @@ contract MyToken is ERC20, ERC20Permit, ERC20Votes, ERC20Wrapper {
 }
 ```
 
-NOTE: Voting power could be determined in different ways: multiple ERC20 tokens, ERC721 tokens, sybil resistant identities, etc. All of these options are potentially supported by writing a custom Votes module for your Governor. The only other source of voting power available in OpenZeppelin Contracts currently is xref:api:token/ERC721.adoc#ERC721Votes[`ERC721Votes`].
+NOTE: Voting power could be determined in different ways: multiple ERC20 tokens, ERC721 tokens, sybil resistant identities, etc. All of these options are potentially supported by writing a custom Votes module for your Governor. The other sources of voting power available in OpenZeppelin Contracts currently are xref:api:token/ERC721.adoc#ERC721Votes[`ERC721Votes`] and xref:api:token/ERC721Wrapper.adoc#ERC721Wrapper[`ERC721Wrapper`].
```

Comment:
No, `ERC721Wrapper` does not provide the voting interface that governance need.

You could use a combo of `ERC721Wrapper` and `ERC721Votes` to wrap non-voting NFTs into voting NFTs ... but the voting part will come from inheriting `ERC721Votes`

Reply:
Thanks for bringing it up! In fact, I want to find a place in the governance docs to share visibility about it.

I see there's a section for ERC20Wrapper [here](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/77fcda8f09b5fc7c72d5f91cdb3b38bce52a0c7c/docs/modules/ROOT/pages/governance.adoc#setup) but includes a code example

```markdown
If your project already has a live token that does not include ERC20Votes and is not upgradeable, you can wrap it in a governance token by using ERC20Wrapper. This will allow token holders to participate in governance by wrapping their tokens 1-to-1.

<Code>
```

I think a code example for ERC721Wrapper is way too much, so what about this?:

```suggestion
NOTE: Voting power could be determined in different ways: multiple ERC20 tokens, ERC721 tokens, sybil resistant identities, etc. All of these options are potentially supported by writing a custom Votes module for your Governor. The only other source of voting power available in OpenZeppelin Contracts currently is xref:api:token/ERC721.adoc#ERC721Votes[`ERC721Votes`] (optionally wrapped in xref:api:token/ERC721Wrapper.adoc#ERC721Wrapper[`ERC721Wrapper`]).
```

Reply:
I would say.

The only other source of voting power available in OpenZeppelin Contracts currently is xref:api:token/ERC721.adoc#ERC721Votes[`ERC721Votes`]. NFT tokens that don't provide this functionnality can be wrapped into voting token isn't a combination of xref:api:token/ERC721.adoc#ERC721Votes[`ERC721Votes`] and xref:api:token/ERC721Wrapper.adoc#ERC721Wrapper[`ERC721Wrapper`].

Reply:
Added with [minor spelling changes](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3863/commits/24ac3dc5f09949532ec843bb899dfd891663be4f)